SignalShareExtension/ShareViewController.swift

```diff
@@ -85,13 +88,95 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate {
         NotificationCenter.default.removeObserver(self)
     }
 
+    private func activate() {
+        Logger.debug("\(self.logTag()) \(#function)")
+
+        // We don't need to use "screen protection" in the SAE.
+
+        ensureRootViewController()
+
+        // Always check prekeys after app launches, and sometimes check on app activation.
+        TSPreKeyManager.checkPreKeysIfNecessary()
+
+        // We don't need to use RTCInitializeSSL() in the SAE.
```

Comment:
I left in explicit comments everywhere I skipped some "work" that the main app does on startup.